docs/topics/dokka-gradle-configuration-options.md

```diff
@@ -0,0 +1,871 @@
+        
+        // Output directory for additional files
+        // Use this block when you want to change the output directory and include extra files
+        outputDirectory.set(rootDir.resolve("docs/api/0.x"))
+        
+        // Use fileTree to add multiple files
+        includes.from(
+            fileTree("docs") {
+                include("**/*.md")
+            }
+        )
+    }
+}
+```
+
+For more information about working with files, see the [Gradle docs](https://docs.gradle.org/current/userguide/working_with_files.html#sec:file_trees).
+
+</tab>
+<tab title="Kotlin custom plugin" group-key="kotlin custom">
+
+```kotlin
+// CustomPlugin.kt
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.jetbrains.dokka.gradle.DokkaExtension
+
+abstract class CustomPlugin : Plugin<Project> {
+    override fun apply(project: Project) {
+        project.plugins.apply("org.jetbrains.dokka")
+
+        project.extensions.configure(DokkaExtension::class.java) { dokka ->
+            
+            dokka.moduleName.set(project.name)
+            dokka.moduleVersion.set(project.version.toString())
+
+            dokka.dokkaPublications.named("html") { publication ->
+                // Standard output directory for HTML documentation
+                publication.outputDirectory.set(project.layout.buildDirectory.dir("dokka/html"))
+                publication.failOnWarning.set(true)
+                publication.suppressInheritedMembers.set(true)
+                publication.offlineMode.set(false)
+                publication.suppressObviousFunctions.set(true)
+                publication.includes.from("packages.md", "extra.md")
+
+                // Output directory for additional files
+                // Use this block when you want to change the output directory and include extra files
+                html.outputDirectory.set(project.rootDir.resolve("docs/api/0.x"))
+                html.includes.from(project.layout.projectDirectory.file("README.md"))
+            }
+        }
+    }
+}
+```
+
+</tab>
+<tab title="Gradle Groovy DSL" group-key="groovy">
+
+```groovy
+plugins {
+    id 'org.jetbrains.dokka' version '%dokkaVersion%'
+}
+
+dokka {
+    dokkaPublications {
+        html {
+            // Sets general module information
+            moduleName.set(project.name)
+            moduleVersion.set(project.version.toString())
+
+            // Standard output directory for HTML documentation
+            outputDirectory.set(layout.buildDirectory.dir("dokka/html"))
+
+            // Core Dokka options
+            failOnWarning.set(false)
+            suppressInheritedMembers.set(false)
+            offlineMode.set(false)
+            suppressObviousFunctions.set(true)
+            includes.from(files("packages.md", "extra.md"))
+
+            // Output directory for additional files
+            // Use this block when you want to change the output directory and include extra files
```

Comment:
Also cut off on the page. (For "Kotlin custom plugin" doesn't matter as much because lines are long in general.)